sdk/core/core/src/az_span.c

```diff
@@ -453,23 +419,26 @@ static AZ_NODISCARD az_result _az_span_builder_append_uint64(az_span* self, uint
   return az_span_append(*self, az_span_init(&value_to_append, 1, 1), self);
 }
 
-AZ_NODISCARD az_result az_span_append_uint64(az_span* self, uint64_t n)
+AZ_NODISCARD az_result
+az_span_append_u64toa(az_span destination, uint64_t source, az_span* out_span)
```

Comment:
Why is this signature split between two lines? Is there a length cut-off that we use as part of the coding style guidelines?

Reply:
Yes there is and we have some tools that reformat the code to a bunch of code style guidelines we have set up. Ask Victor, Anton, or Rick about them.

Reply:
We use Clang formating of 100 characters.  The exact length was rather hotly contested but at the end the consensus was 100 was the standard we settled on in the guidelines
Take a look in the root folder ".clang-format" for our auto format settings